view/script/dashboard.js

```diff
@@ -4,6 +4,14 @@
     var nameColumn = 0;
     var table = $('#trials').DataTable({order: [nameColumn, 'desc']});
     var strategy = filterStrategyFactory();
+    var selectedChoice = null;
+
+    $('.dropdown-menu').on('click', 'div', function selectItem () {
+        selectedChoice = $(this).text().trim();
```

Comment:
remove `var selectedChoice = null` on line 7.
and add `var` in front of line 10.